### PR TITLE
upgrade to go1.19

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- upgrade to go1.19

### CODE REVIEW

**Comment by cpanato** [comment]
getting this error 

```
2022-08-31T09:31:04.580156925Z stderr F {"severity":"fatal","ts":1661938264.5799956,"caller":"app/serve.go:221","message":"x509: certificate contains duplicate extensions","stacktrace":"github.com/sigstore/fulcio/cmd/app.runServeCmd\n\tgithub.com/sigstore/fulcio/cmd/app/serve.go:221\ngithub.com/spf13/cobra.(*Command).execute\n\tgithub.com/spf13/cobra@v1.5.0/command.go:876\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\tgithub.com/spf13/cobra@v1.5.0/command.go:990\ngithub.com/spf13/cobra.(*Command).Execute\n\tgithub.com/spf13/cobra@v1.5.0/command.go:918\ngithub.com/spf13/cobra.(*Command).ExecuteContext\n\tgithub.com/spf13/cobra@v1.5.0/command.go:911\ngithub.com/sigstore/fulcio/cmd/app.Execute\n\tgithub.com/sigstore/fulcio/cmd/app/root.go:37\nmain.main\n\tgithub.com/sigstore/fulcio/main.go:25\nruntime.main\n\truntime/proc.go:250"}
```

**Review by haydentherapper — DISMISSED** [review]
To confirm,  the go.mod version had to be updated after this has been submitted?

**Comment by cpanato** [comment]
> To confirm, the go.mod version had to be updated after this has been submitted?

yes

File: .github/workflows/verify.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ jobs:
       - uses: actions/setup-go@268d8c0ca0432bb2cf416faae41297df9d262d7f # v3.2.0
         with:
           go-version: ${{ env.GOVERSION }}
+          check-latest: true
```
Comment (haydentherapper) [inline .github/workflows/verify.yml:34]: This needs to be removed, correct?

Comment (cpanato) [reply .github/workflows/verify.yml:34]: yes

**Comment by cpanato** [comment]
all green now PTAL @dlorenc @haydentherapper @bobcallaway

**Review by bobcallaway — APPROVED** [review]

**Review by haydentherapper — APPROVED** [review]

**Pull request merged**